action.yml

```diff
@@ -0,0 +1,15 @@
+name: "Install ispc"
+description: "Install Intel ispc."
+inputs:
+  version:
+    description: "Release version of ispc to install."
+    required: false
+  platform:
+    description: "Platform to download release of ispc for."
+    required: true
+  architecture:
+    description: "Architecture to download relase of ispc for, on MacOS and Linux platforms."
+    required: false
+runs:
+  using: 'node20'
+  main: 'build/main.cjs'
```

Comment:
I'm still not sure why it is `build/main.cjs` and not just `src/main.js` as referred un the docs: [build/main.cjs](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs-for-javascript-actions)

Reply:
That's an interesting point. I'm looking at example repo with JS GH action and it indeed runs compiled version of JS: see [here](https://github.com/actions/hello-world-javascript-action/blob/8177c4be451a6331f95385502f9c01458cc6346c/action.yml#L19C3-L19C22).

I'm not familiar with how JS distribution works, but seems that it's a common practice. Though for any security reviewer the compiled JS code might cause a heart attack.

Reply:
Yeah, I'd like to get rid of the built source as well. But actions shouldn't really pull node modules on execution, and I'm not sure actions support a different artifact distribution mechanism. Let me know if you find anything.

Reply:
It's good that we spelled it out.

We are not JS experts, so things like that need clarification. I don't know how reproducible the build is, but possibly we need `DEVELOPERS.md` file with a recipe how to reproduce the build and update when needed.